// discordjs-colors/lib/index.js
export function defaultColor() { return "#000000" }
export function white() { return "#FFFFFF" }
export function aqua() { return "#1ABC9C" }
export function green() { return "#2ECC71" }
export function blue() { return "#3498DB" }
export function purple() { return "#9B59B6" }
export function luminousvividpink() { return "#E91E63" }
export function gold() { return "#F1C40F" }
export function orange() { return "#E67E22" }
export function red() { return "#E74C3C" }
export function gray() { return "#95A5A6" }
export function navy() { return "#34495E" }
export function darkaqua() { return "#11806A" }
export function darkgreen() { return "#1F8B4C" }
export function darkblue() { return "#206694" }
export function darkpurple() { return "#71368A" }
export function darkvividpink() { return "#AD1457" }
export function darkgold() { return "#C27C0E" }
export function darkorange() { return "#A84300" }
export function darkred() { return "#992D22" }
export function darkgray() { return "#979C9F" }
export function darkergray() { return "#7F8C8D" }
export function lightgray() { return "#BCC0C0" }
export function darknavy() { return "#2C3E50" }
export function blurple() { return "#7289DA" }
export function grayple() { return "#99AAB5" }
export function darkbutnotblack() { return "#2C2F33" }
export function notquiteblack() { return "#23272A" }
export function randomColor() { return "#"+(0x1000000+(Math.random())*0xFFFFFF).toString(16).substring(1, 7).toUpperCase() }

module.exports = defaultColor();
